Kernel/Process.cpp

```diff
@@ -2726,6 +2727,28 @@ int Process::sys$reboot()
     return ESUCCESS;
 }
 
+int Process::sys$mount(int devmajor, int devminor, const char* mountpoint)
+{
```

Comment:
The very first thing to do in this function is validate the `mountpoint` pointer since it comes from userspace:

```cpp
if (!validate_read_str(mountpoint))
    return -EFAULT;
```